cmd/init/config.go

```diff
@@ -0,0 +1,68 @@
+package init
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	initConfig "github.com/flyteorg/flytectl/cmd/config/subcommand/init"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	cmdUtil "github.com/flyteorg/flytectl/pkg/commandutils"
+	"github.com/flyteorg/flytectl/pkg/util"
+)
+
+const (
+	initConfigCmdShort = "Used for generating config template."
+	initConfigCmdLong  = `init config will create a config in flyte directory i.e ~/.flyte
+Generate sandbox config.
```

Comment:
Can we link to what does it mean to have a sandbox https://docs.flyte.org/en/latest/deployment/sandbox.html
and also configuration related to it.